tests/test_cq.py

```diff
@@ -1,5 +1,6 @@
 # SPDX-License-Identifier: (GPL-2.0 OR Linux-OpenIB)
 # Copyright (c) 2019 Mellanox Technologies, Inc. All rights reserved. See COPYING file
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All rights reserved.
```

Comment:
Is it possible to have 2 copyright statements?

Reply:
Yes, the copyright statements are not limited to one company/author. Everyone who adds/changes code can claim their copyright on it.

Of course in this particular case where Gal changed two lines and requested copyright for that, I'm feeling sad, it has zero value outside of the micro-world of enterprise lawyers.

Thanks